cmd/tiller-proxy/internal/handler/handler.go

```diff
@@ -233,7 +233,7 @@ func (h *TillerProxy) UpgradeRelease(w http.ResponseWriter, req *http.Request, p
 
 // ListAllReleases list all releases that Tiller stores
 func (h *TillerProxy) ListAllReleases(w http.ResponseWriter, req *http.Request) {
-	apps, err := h.ProxyClient.ListReleases("", h.ListLimit)
+	apps, err := h.ProxyClient.ListReleases("", h.ListLimit, req.URL.Query().Get("status"))
```

Comment:
Should we call the parameter "statuses" since you can add multiple comma separated?

Reply:
I agree with calling them `statuses`